MAUI/StepProgressBar/description.md

```diff
@@ -0,0 +1,413 @@
+                    LabelSpacing="12"
+                    ActiveStepIndex="1"
+                    ActiveStepProgressValue="50"
+                    ItemsSource="{Binding StepProgressItem}">
+</stepProgressBar:SfStepProgressBar>                                                                                             
+
+<ContentPage.BindingContext>
+    <local:ViewModel />
+</ContentPage.BindingContext>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="10" %}
+
+ViewModel viewModel = new ViewModel();
+SfStepProgressBar stepProgressBar = new SfStepProgressBar()
+{
+    VerticalOptions = LayoutOptions.Center,
+    HorizontalOptions = LayoutOptions.Center,
+    Orientation = StepProgressBarOrientation.Vertical,
+    LabelSpacing = 12,
+    ActiveStepIndex = 1,
+    ActiveStepProgressValue = 50,
+    ItemsSource = viewModel.StepProgressItem,
+};
+
+this.Content = stepProgressBar;
+
+{% endhighlight %}
+{% endtabs %}
+
+N> If both `PrimaryText` and `PrimaryFormattedText` are provided, then `PrimaryFormattedText` will be displayed. This precedence as same as for the `SecondaryText` and `SecondaryFormattedText`.
+
+## Customize description
+Using [LabelPosition]() and [LabelSpacing]() property, the description alignment and space between the description and step can be customized, respectively. The following code example explains how to customize the LabelPosition and LabelSpacing properties.
+{% tabs %}
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="6 7" %}
+
+<stepProgressBar:SfStepProgressBar
+                        x:Name="stepProgress"
+                        VerticalOptions="Center"
+                        HorizontalOptions="Center"                                        
+                        Orientation="Vertical"                                                                                   
+                        LabelSpacing="28"
+                        LabelPosition="Bottom">
+</stepProgressBar:SfStepProgressBar> 
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="6 7" %}
+
+SfStepProgressBar stepProgressBar = new SfStepProgressBar()
+{
+    VerticalOptions = LayoutOptions.Center,
+    HorizontalOptions = LayoutOptions.Center,
+    Orientation = StepProgressBarOrientation.Vertical,
+    LabelSpacing = 28,
+    LabelPosition = LabelPosition.Bottom,
+};
+
+this.Content = stepProgressBar;
+
+{% endhighlight %}
+{% endtabs %}
+
+## StepProgressBar Image Customization
+
+The [StepProgressBar]() control provides support for adding images as the content of the step. You can add the images by giving image path and with font icons.
+
+### Customization using Image
+
+You can add the images by giving image path to the [ImageSource]() property in `SfStepProgressBar`. The following code example explains how to set an image in step view.
+
+#### Initialize view model
+
+* Create a simple Observable Collection of item with image source for the step progressbar,
```

Comment:
* Create a simple Observable Collection of item with image source for the step progressbar.

Reply:
Content modified